### ISSUE TITLE
Rev 3 Multiplexer kills RasPI3

### ISSUE THREAD

**Issue by jtice01** [ivmech/ivport#17]
When I connect the ribbon cable to the Rasperry Pi 3 it shuts off. Just having the multiplexer connected to the GPIO headers doesnt cause it to fail. Only when I connect the ribbon cable from the Input to Camera port.
Defective board?

**Comment by ivmech** [comment]
Hi, can you send some photos of board and configuration?

**Comment by jtice01** [comment]
Actually, I discovered the problem. It was a bad ribbon cable. I can now boot Raspberry Pi 3 with multiplexer.

**Comment by ivmech** [comment]
Yes, you should use same type ribbon cable with camera modules, 15pin anisotropy,

**Comment by jasdeepdhillon13** [comment]
i had the same issue and tried 15 pin anisotropy ribbon cables
